Development

epinczinger opened the pull request:
The project was built mainly with bootstrap for responsive design. The aim was to copy the original [NewsWeek](https://www.newsweek.com/) page. It has a small size view, a medium and a large one.

epinczinger committed:
mobile main section at 20%

epinczinger committed:
heroes pandemic and culture travel section done

epinczinger committed:
done features section

epinczinger committed:
main mobile section done

epinczinger committed:
in the magazine section done

epinczinger committed:
mobile size done, fighting agains ordering md size

epinczinger committed:
fighting against ordering

epinczinger committed:
fix stylelint errors

epinczinger committed:
fix href and stylelint

epinczinger committed:
delete reset.css

epinczinger committed:
update readme

epinczinger committed:
screenshot and readme updated

epinczinger committed:
done sticky divs

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

addod19 reviewed (changes requested):
## Status: Require changes :recycle: :traffic_light: 

Hi @epinczinger, awesome job with this project :+1: UI looking great, great commit messages, README professional :slightly_smiling_face: However, in order to get our project approved, we need to address the comments below:
- Kindly enable stickler by visiting stickler-ci.com to check for style errors/warnings
- ```Mobile device 605px```, let us fix the positioning by comparing our site to the source
``` clone ```
![605px](https://user-images.githubusercontent.com/46388113/84311957-b6755700-ab53-11ea-8ed7-a3f9e8a9a5ed.png)
``` source ```
![605pxS](https://user-images.githubusercontent.com/46388113/84311970-be34fb80-ab53-11ea-8359-b67af2536846.png)
``` US, Sports, Culture, Tech and Science, Business, World  ``` sections, let us align the elements
``` clone ```
![sprtheight](https://user-images.githubusercontent.com/46388113/84312418-6b0f7880-ab54-11ea-975f-33c5d10c91da.png)
``` source ```
![sprtHeight1](https://user-images.githubusercontent.com/46388113/84312467-82e6fc80-ab54-11ea-88da-b8e737c28b95.png)
- Let add a little margin around the ```Editors Pick``` so it looks like the source
``` clone ```
![edpicks](https://user-images.githubusercontent.com/46388113/84313147-a3638680-ab55-11ea-9bde-62eb86643e49.png)
``` source ```
![edpick1](https://user-images.githubusercontent.com/46388113/84313201-b8d8b080-ab55-11ea-9726-ec3367474b37.png)
- Let us add a gray background to the See All Features and increase the height of our images a little and the date
``` clone ```
![seeFeatures](https://user-images.githubusercontent.com/46388113/84314042-f12cbe80-ab56-11ea-8a04-04b50e8db79f.png)
``` source ```
![seeFeatures1](https://user-images.githubusercontent.com/46388113/84314073-fbe75380-ab56-11ea-8c1a-a36a758a1151.png)

Kindly fix these comments and send another code review :smiley: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

epinczinger committed:
fix tse requirements; footer justify, date backgrounds, and alignments

epinczinger commented:
Hi, @addod19 thanks for your review! I've fixed the changes you pointed. I'm submitting for another check.
Kind regards

BrittanyBlake commented:
### Pull request under review by Brittany Blake (TSE)

BrittanyBlake reviewed (changes requested):
### Status: Changes Required ♻️ 
****
### Hi @epinczinger ! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] Please take a look at your footer once again in the `605 px` screen size. The layout is still in need of improvement as was requested by the previous TSE. 
Your clone 👇 
![Screen Shot 2020-06-11 at 2 30 55 AM](https://user-images.githubusercontent.com/48804258/84332411-281dc700-ab8d-11ea-8419-ff4d0f49458b.png)
The original page 👇 
![Screen Shot 2020-06-11 at 2 30 46 AM](https://user-images.githubusercontent.com/48804258/84332414-294ef400-ab8d-11ea-949f-5616bbfaad05.png)

- [x] As was requested by the previous TSE, the US, Sports, Culture, Tech and Science, Business, and World sections should be aligned. Please compare your clone with the original page and fix this.
Your clone 👇 
![Screen Shot 2020-06-11 at 2 33 12 AM](https://user-images.githubusercontent.com/48804258/84332544-77fc8e00-ab8d-11ea-8b64-be83e61564f9.png)
The original page 👇 
![Screen Shot 2020-06-11 at 2 32 51 AM](https://user-images.githubusercontent.com/48804258/84332546-792dbb00-ab8d-11ea-8065-175ff1012bc1.png)

- [x] The date on the "see all features" should be on the magazine cover, not below it. Additionally, this section is not aligned properly. Please compare your clone with the original page and fix this. 
Your clone 👇 
![Screen Shot 2020-06-11 at 2 35 53 AM](https://user-images.githubusercontent.com/48804258/84332718-f1947c00-ab8d-11ea-8d65-c3ab4dba7db4.png)
The original page 👇 
![Screen Shot 2020-06-11 at 2 36 10 AM](https://user-images.githubusercontent.com/48804258/84332722-f2c5a900-ab8d-11ea-8e90-0adcd74c43bc.png)

- [x] In the "in the magazine section" on the medium screen size `(768 px)` the title of the magazines should be in one line. 
Your clone 👇 
![Screen Shot 2020-06-11 at 2 35 18 AM](https://user-images.githubusercontent.com/48804258/84332829-3fa97f80-ab8e-11ea-9917-41d082a760fb.png)
The original page 👇 
![Screen Shot 2020-06-11 at 2 35 38 AM](https://user-images.githubusercontent.com/48804258/84332831-40daac80-ab8e-11ea-8be6-33928ff3333d.png)





****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

epinczinger committed:
tse review changes; footer and in the magazine section

epinczinger committed:
max-width main

epinczinger commented:
Hi! I've fixed the changes required. I think the live demo link you opened wasn't correctly updated because I had fixed the alignments and the all features date before.
I've tried to refresh it so I hope you can see it properly now. Sorry for the inconvenient.

Kind Regards

(I'm not sure why sometimes on the live demo link some images are not displayed properly)

epinczinger committed:
date font size changed

epinczinger committed:
max-height to features img

epinczinger committed:
update style.css

tirthajyoti-ghosh commented:
# Pull Request under review by T. Ghosh (TSE)

tirthajyoti-ghosh reviewed (changes requested):
# Status: Changes required 🚨

## Impressive project @epinczinger ✨👏, awesome README btw ✨

### However, your project still need some changes in order to get it approved.

- The `navbar` needs to have spaces on either side.

   Right now 👇

   ![Annotation 2020-06-11 233852](https://user-images.githubusercontent.com/57726348/84423952-f2381b80-ac3c-11ea-9d41-06f68dbdbbe7.jpg)

   Original 👇

   ![Annotation 2020-06-11 234059](https://user-images.githubusercontent.com/57726348/84423996-0da32680-ac3d-11ea-93f1-01f0038d7e11.jpg)

- Increase the width of the `top story` section and decrease the width `the debate/ opinion` section
  
  ![Annotation 2020-06-11 234239](https://user-images.githubusercontent.com/57726348/84424216-6377ce80-ac3d-11ea-888b-bbcd891d518b.jpg)

- Increase the width of the image in `sign-up`
  
  ![Annotation 2020-06-11 234551](https://user-images.githubusercontent.com/57726348/84424493-d08b6400-ac3d-11ea-8221-2b107562ec55.jpg)

- Bottom-left tags are missing in `editor's picks` section

   Right now 👇

   ![Annotation 2020-06-11 234740](https://user-images.githubusercontent.com/57726348/84424630-04ff2000-ac3e-11ea-9d00-952a3884da56.jpg)

   Original 👇

   ![Annotation 2020-06-11 234908](https://user-images.githubusercontent.com/57726348/84424745-29f39300-ac3e-11ea-88bd-2c97d487ed49.jpg)

-  Entire `subscribe/newsletter` section is missing

   Right now 👇

   ![Annotation 2020-06-11 235046](https://user-images.githubusercontent.com/57726348/84424954-6e7f2e80-ac3e-11ea-8db5-15286fceb3e9.jpg)

   Original 👇

   ![Annotation 2020-06-11 235058](https://user-images.githubusercontent.com/57726348/84424984-7939c380-ac3e-11ea-9464-fdafc79c1e2f.jpg)

- The footer must also have space on either side.

   Right now 👇
   
   ![Annotation 2020-06-11 235209](https://user-images.githubusercontent.com/57726348/84425093-a4bcae00-ac3e-11ea-9148-1e9f2e5f5c97.jpg)

   Original 👇
  
   ![Annotation 2020-06-11 235220](https://user-images.githubusercontent.com/57726348/84425115-ac7c5280-ac3e-11ea-8b34-f380104fff15.jpg)

I have also left some comments below, be sure to check them.

tirthajyoti-ghosh commented on assets/style.css:
```diff
@@ -0,0 +1,263 @@
+
+a {
+  color: inherit;
+  text-decoration: none !important;
```
- Remove `!important`.

tirthajyoti-ghosh commented on assets/style.css:
```diff
@@ -0,0 +1,263 @@
+}
+
+img[src="assets/images/n.jpeg"] {
+  width: 29px !important;
```
- Remove `!important`.

tirthajyoti-ghosh commented on index.html:
```diff
@@ -0,0 +1,1057 @@
+
+    <!-- PRINCIPAL NAVBAR  -->
+
+    <nav class="navbar sticky-top py-lg-3" style="background-color: red;">
```
- Remove in-line styles, here and in all other places. It's a CSS bad practice.

tirthajyoti-ghosh reviewed (commented):
Apply these changes and request for another code review.

Happy coding 👨‍💻

epinczinger committed:
subscribe and newsletter added

epinczinger committed:
style fix

epinczinger committed:
important erased

epinczinger commented:
Hi! Changes have been done. Please check the current version because the live link seems to have a delay. 
Kind regards.

peterrobert commented:
### PULL REQUEST UNDER REVIEW BY PETER ROBERT (TSE)

peterrobert reviewed (approved):
### APPROVED :+1: :+1: :100: 
**GREAT WORK IMPROVING ON ALL THE REQUIRED CHANGES FROM THE PREVIOUS TSE** :+1: 
Your project looks great :+1:  

**[OPTIONAL]** 
You can make the search bar look like the original. Yours :arrow_down: 
![search](https://user-images.githubusercontent.com/32361963/84449448-223fe880-ac56-11ea-8e96-0d97782d27a9.jpg)
original :arrow_down: 
![or](https://user-images.githubusercontent.com/32361963/84449507-4c91a600-ac56-11ea-8642-f251b6f7e229.jpg)

**ONCE YOU MAKE THAT CHANGE KINDLY MERGE THE BRANCH AND SUBMIT THE COMPLETION FORM** :1st_place_medal: 

### HAPPY CODING :computer:

epinczinger commented:
Thanks! I might have changed the colour when modified the CSS, but I'll fix it. Regards
